experiment/cherrypicker/server.go

```diff
@@ -165,15 +177,17 @@ func (s *Server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent
 	targetBranch := cherryPickMatches[0][1]
 
 	if ic.Issue.State != "closed" {
-		// Only members should be able to do cherry-picks.
-		ok, err := s.ghc.IsMember(org, commentAuthor)
-		if err != nil {
-			return err
-		}
-		if !ok {
-			resp := fmt.Sprintf("only [%s](https://github.com/orgs/%s/people) org members may request cherry picks. You can still do the cherry-pick manually.", org, org)
-			s.log.WithFields(l.Data).Info(resp)
-			return s.ghc.CreateComment(org, repo, num, plugins.FormatICResponse(ic.Comment, resp))
+		if !s.allowAll {
+			// Only members should be able to do cherry-picks.
```

Comment:
/nit mode on

so this isn't anybody, just members right? ;-)

Reply:
nvmd. read this wrong.

Reply:
;)